components/ca-crud.css

```diff
@@ -0,0 +1,90 @@
+ca-crud {
+    display: block;
+    position: relative;
+}
+
+ca-crud h2 {
+    font-size: 1.3em;
+    margin-bottom: 8px;
+}
+
+ca-crud h3 {
+    font-size: 1.1em;
+    margin: 6px 0;
+}
+
+ca-crud.form-view ca-list, ca-crud.form-view button {
+    visibility: hidden;
+    display: none;
+}
+
+ca-crud .buttons {
+    text-align: left;
+}
+
+ca-crud .buttons #btnScroll {
+    float: right;
+}
+
+ca-crud button {
+    color: #fff;
+    cursor: pointer;
+    background-color: #333;
+    text-shadow: -1px -1px 0 rgba(0, 0, 0, 0.5);
+    border: 1px solid #333;
+    border-color: #666;
+    border-color: rgba(0, 0, 0, 0.6);
+    border-radius: 5px;
+    -webkit-box-shadow: none;
+    box-shadow: none;
+    margin: 10px 0;
+    width: 100px;
+    padding: 0.5em;
+}
+
+ca-crud[data-action-complete]:before {
+    content: "The following action complete " attr(data-action-complete);
```

Comment:
Does this make sense as a sentence?  Not sure I understand the meaning of the full sentence.